src/NServiceBus.AcceptanceTests/Correlation/When_using_a_custom_correlation_id.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace NServiceBus.AcceptanceTests.Correlation
+﻿#pragma warning disable 618
```

Comment:
I'd prefer to see this scoped around the specific call that needs it (line 19), instead of turning this off for the entire file.